app/lib/url.rb

```diff
@@ -1,26 +1,39 @@
-module App
-  module_function
-
-  def protocol
+# Utilities methods to safely build app wide URLs
+module URL
```

Comment:
I feel like a name such as `UrlBuilder` would be safer and even more clear here but I am game for this. At least it's more clear than before.